static/js/lib/auth.js

```diff
@@ -15,11 +17,8 @@ const renewSession = async () => {
 
 const redirectAndReject = async (reason: string) => {
   // redirect to the authenticating app
-  if (SETTINGS.auth_url) {
-    window.location = SETTINGS.auth_url
-    return Promise.reject(reason)
-  }
-  return Promise.reject("Unable to redirect to authenticating app")
+  window.location = authRequiredURL()
+  return Promise.reject(reason)
```

Comment:
do we still want to check for `SETTINGS.auth_url` and redirect there if it's present?

Reply:
I don't think so. At this point the code tried to get a new token but failed for some reason. The `auth_url` set in micromasters should be `/discussions/` on its site. I don't think there's a circumstance where the user fails to refresh the JWT but is also still logged into micromasters.

@rhysyngsun can you confirm?

Reply:
That makes sense, the only thing that occurs to me is if MM drops the JWT refresh request for some reason, but in that case we would probably just want to go to the `AUTH_REQUIRED_URL` anyway.

Reply:
There's not a normal use case a user is logged in but can't refresh the token, but there could be some abnormal ones (e.g. `DiscussionUser` gets deleted or `DiscussionUser.username` gets nulled out).

Reply:
In either of those cases we lose the connection between MM and OD for that user. When they log in again they will get a new user account. I think it's not worth handling this beyond redirecting the user to the auth_required page

Reply:
Makes sense!